lib/connextclient/ConnextClient.ts

```diff
@@ -591,6 +591,11 @@ class ConnextClient extends SwapClient {
       assetId,
       amount: BigInt(units).toString(),
     });
+    this.sendRequest('/request-collateral', 'POST', {
```

Comment:
Maybe also add a info logging here to notify when the `/request-collateral` request completes?

Reply:
That would be quite helpful indeed.